nixos/modules/services/system/earlyoom.nix

```diff
@@ -127,8 +136,11 @@ in
 
     extraArgs = mkOption {
       type = types.listOf types.str;
-      default = [];
-      example = [ "-g" "--prefer '(^|/)(java|chromium)$'" ];
+      default = [ ];
+      example = [
+        "-g"
+        "--prefer '(^|/)(java|chromium)$'"
```

Comment:
This format seems to no longer work due to the call to `lib.escapeShellArgs` that wasn't previously there. I had to split out space-separated flags into individual list items: https://github.com/msfjarvis/dotfiles/commit/a711550e4e38256a2834a40a893fddbab206c359

Without the change in my configuration, the flag that ended up in `EARLYOOM_ARGS` was `'--avoid '\\''^(gnome.*|firefox.*|pipewire.*|git.*)$'\\'''` which was seen by earlyoom as a single parameter not a flag and caused the service to stop running.

Reply:
@msfjarvis Thanks, finally was looking into why my earlyoom service hasn't worked for a few days and saw this PR.

Reply:
@misuzu this needs a release note

Reply:
@oxalica this needs a release note